packages/jira-adapter/src/filters/portal_settings.ts

```diff
@@ -0,0 +1,89 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Change, InstanceElement, getChangeData, isAdditionChange, isAdditionOrModificationChange, isInstanceChange } from '@salto-io/adapter-api'
+import { getParent } from '@salto-io/adapter-utils'
+import _ from 'lodash'
+import { deployChanges } from '../deployment/standard_deployment'
+import { FilterCreator } from '../filter'
+import { PORTAL_SETTINGS_TYPE_NAME } from '../constants'
+import JiraClient from '../client/client'
+
+const deployPortalSettings = async (
+  change: Change<InstanceElement>,
+  client: JiraClient,
+): Promise<void> => {
+  const portalSettings = getChangeData(change)
+  const parent = getParent(portalSettings)
+  if (isAdditionOrModificationChange(change)) {
+    const nameUrl = `/rest/servicedesk/1/servicedesk-data/${parent.value.key}/name`
+    if (isAdditionChange(change) || change.data.before.value.name !== change.data.after.value.name) {
+      await client.put({
+        url: nameUrl,
+        data: { name: portalSettings.value.name },
+      })
+    }
+    if (isAdditionChange(change) || change.data.before.value.description !== change.data.after.value.description) {
+      await client.put({
+        url: `/rest/servicedesk/1/servicedesk-data/${parent.value.key}/desc`,
+        data: { description: portalSettings.value.description },
+      })
+    }
+    if (isAdditionChange(change)
+    || change.data.before.value.announcementSettings.canAgentsManagePortalAnnouncement
+        !== change.data.after.value.announcementSettings.canAgentsManagePortalAnnouncement) {
```

Comment:
`announcementSettings` might be undefined

```suggestion
    || change.data.before.value.announcementSettings?.canAgentsManagePortalAnnouncement
        !== change.data.after.value.announcementSettings?.canAgentsManagePortalAnnouncement) {
```